package.json

```diff
@@ -7,19 +7,32 @@
     "author": "CXL <leho@cxl.com>",
     "license": "MIT",
     "peerDependencies": {
-        "eslint": "^7.7.0",
-        "husky": "^4.2.5",
-        "lint-staged": "^10.2.13",
-        "prettier": "^2.1.1"
+        "eslint": "^6.8.0",
```

Comment:
We probably don't want to go backwards in eslint versions?

Reply:
Yes **10up** packages installed these backwards versions... by default...
I left it alone for initial testing, definitely give it a try with latest versions and see if it still works.